plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -162,4 +164,54 @@ public static function get_automated_support_countries() {
 
 		return $tax_supported_countries;
 	}
+
+	/**
+	 * Get an array of countries that support Stripe tax.
+	 *
+	 * @return array
+	 */
+	private static function get_stripe_tax_support_countries() {
+		// https://docs.stripe.com/tax/supported-countries#supported-countries accurate as of 2024-08-26.
+		// countries with remote sales not included.
+		return array(
```

Comment:
From the [docs](https://docs.stripe.com/tax/supported-countries#supported-countries), it seems Ukraine `UA` is missing in our list. However, it's also missing from the screenshot in p7fD6U-eHc-p2#comment-28712.

Should we include Ukraine?